### PR TITLE
Fix/inkwidget preview

### LINKED ISSUE #1568: Broken WidgetPreview
**Describe the bug**
trying to use the widgetpreview of large inkwidgets will show nothing and throw an out of memory exception:
[2/14/2024 12:28:51 PM] [Error    ] Unhandled exception in WolvenKit v8.12.3.0
[2/14/2024 12:28:51 PM] [Error    ] Message: A Task's exception(s) were not observed either by Waiting on the Task or accessing its Exception property. As a result, the unobserved exception was rethrown by the finalizer thread. (Out of memory.)
Source: 
StackTrace: 

based on my understanding inkwidget files are a main root container and several other library items and these library items are all loaded via script and are not all added to the main root widget and trying to load all of them together may be the cause of out of memory my suggestion would be to separate every library item and make a dropdown selection(like appearance selection in meshpreview) to select different library items and loading them separately

**To Reproduce**
1. open a large inkwidget file(i used ripperdoc.inkwidget)
2. select widgetpreview
3. wait

**Expected behavior**
showing the widgetpreview

**Version (please complete the following information):**
 - Version 8.12.3.0

**Screenshots**
image
image

### LINKED ISSUE #1700: Widget Preview broken for some inkwidgets
**Describe the bug**
The "Widget Preview" feature for inkwidgets is broken for some inkwidgets

**To Reproduce**
1. Open `base\gameplay\gui\widgets\vehicle\herrera\herrera_outlaw.inkwidget` or `base\gameplay\gui\widgets\vehicle\rayfield\rayfield_caliburn.inkwidget` in a wkit project
2. Switch to "Widget Preview"
3. Nothing happens, no preview is loaded. The inkwidget contains information/and is valid as it is used in the vehicle interior UI. Clicking on "Regenerate inkTextWidgets" also does nothing

**Expected behavior**
Inkwidget preview working

I also tested `base\gameplay\gui\quests\q000\aerondight_rayfield\aerondight_rayfield.inkwidget` which straight up crashes wkit

**Version:**
 - Version: can be reproduced in 8.13.0 and Nightly (04-28)

**Screenshots**

image

### PR DESCRIPTION
**Pull request by seberoth**
InkWidget Fixes

**Fixed:**
- Usage of wrong buffer type, when `inkWidgetLibraryItem.Package` is used for preview. Fixes #1700
- Exception while slicing images. Fixes #1568

**Notes:**
Still some issues left, but at least can preview a few more

### CODE REVIEW

File: WolvenKit/Layout/inkWidgets/inkBaseImageControl.cs [hunk 1]
```diff
@@ -177,7 +180,9 @@ protected virtual void DrawImage(SizeF size)
                 {
                     attributes.SetWrapMode(System.Drawing.Drawing2D.WrapMode.TileFlipXY);
 
-                    if (Rect.Top != 0)
+                    DrawNineSlicedImage(gfx, destBitmap, sourceBitmap, attributes);
+
+                    /*if (Rect.Top != 0)
```
Comment (manavortex) [inline WolvenKit/Layout/inkWidgets/inkBaseImageControl.cs:185]: Can this code go?

**Review by manavortex — APPROVED** [review]
LGTM

File: WolvenKit/Layout/inkWidgets/inkBaseImageControl.cs (hunk 1 above)
Comment (seberoth) [reply WolvenKit/Layout/inkWidgets/inkBaseImageControl.cs:185]: Yeah, just kept it for reference if I fucked up somehow

**Review by manavortex — APPROVED** [review]
LGTM (sorry, didn't see that this was still open)

**Pull request merged**